website/cue/reference/components/sources/aws_sqs.cue

```diff
@@ -0,0 +1,125 @@
+package metadata
+
+components: sources: aws_sqs: components._aws & {
+	title: "AWS SQS"
+
+//	features: {
+//		collect: {
+//			checkpoint: enabled: false
+//			tls: {
+//				enabled:                true
+//				can_enable:             true
+//				can_verify_certificate: false
+//				can_verify_hostname:    false
+//				enabled_default:        false
+//			}
+//			from: components._kafka.features.collect.from
+//		}
+//		multiline: enabled: false
+//		codecs: {
+//			enabled:         true
+//			default_framing: "bytes"
+//		}
+//	}
```

Comment:
Intentionally left?

Reply:
This is what happens when I write most of the docs on a Friday, and finish on a Monday :)
Need to figure out what all these features mean and use the correct ones...